pkg/manager/member/pd_member_manager.go

```diff
@@ -537,6 +538,11 @@ func getNewPDSetForTidbCluster(tc *v1alpha1.TidbCluster, cm *corev1.ConfigMap) (
 		pdConfigMap = cm.Name
 	}
 
+	clusterVersionGT4, err := clusterVersionGreaterThan4(tc.PDVersion())
```

Comment:
`s/clusterVersionGT4/clusterVersionGE4`

`GE` implies `Greater than or Equal to`